mission/src/components/ItemInfo/Review.vue

```diff
@@ -1,57 +1,52 @@
 <template>
   <div class="content">
     <div class="product-info info">리뷰</div>
-    <div class="review-card" v-for="(item, index) of reviews" :key="index">
+    <div v-if="reviews[0].review_no === 0" data-test="no-review">{{ STRINGS.NO_REVIEW }}</div>
+    <div
+      v-else
+      class="review-card"
+      v-for="(item, index) of reviews"
+      :key="index"
+      data-no="item.review_no"
+    >
       <div>
-        {{ maskingId(item.user) }}
-        <a class="regist-date">{{ item.registDate }}</a>
+        <a class="writer">{{ item.writer }}</a>
+        <a class="regist-date">{{ item.created }}</a>
       </div>
       <div class="contents">
         <div class="review-info">
           <p class="title">{{ item.title }}</p>
-          {{ item.comment }}
+          <p class="content">{{ item.content }}</p>
+          <br />
+          <a class="likes">
+            <a>좋아요</a>
+            {{ item.likes_count }}
+          </a>
         </div>
-        <div class="image" :style="{ 'background-image': `url('${item.image}')` }"></div>
+
+        <div class="image" :style="{ 'background-image': `url('${item.img}')` }"></div>
       </div>
     </div>
   </div>
 </template>
 
 <script>
+import Strings from '@/utils/js/Strings';
+
 export default {
   name: 'Review',
-  data() {
-    return {
-      reviews: [
-        {
-          user: 'user01',
-          title: '만족해요',
-          comment: '핏이 아주 잘 맞습니다. 대만족-!',
-          /* eslint-disable global-require */
-          image: require('@/assets/img/detail/detail_01.jpg'),
-          registDate: '2022.01.01',
-        },
-        {
-          user: 'likeon022',
-          title: '22만족해요',
-          comment: '22핏이 아주 잘 맞습니다. 대만족-!',
-          /* eslint-disable global-require */
-          image: require('@/assets/img/detail/detail_01.jpg'),
-          registDate: '2022.01.01',
-        },
-      ],
-    };
+  props: {
+    reviews: {
+      type: Array,
+      default: () => [],
```

Comment:
저는 배열의 기본값을 표현할 때 아래와 같이 했는데, 경민님처럼 한 줄로 표현하는 게 훨씬 간단하고 좋아 보이네요-

```suggestion
      default() {
        return [];
      },
```